autoware_adapi_version_msgs/package.xml

```diff
@@ -2,9 +2,10 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>autoware_adapi_version_msgs</name>
-  <version>0.0.0</version>
-  <description>The autoware_adapi_version_msgs package</description>
+  <version>1.1.0</version>
```

Comment:
I think 1.0.0 is appropriate since this package has not changed since the first release.
```suggestion
  <version>1.0.0</version>
```

Reply:
Since we will be hitting the version tag against the repository, we believe that the version of each package must be consistent. Therefore, I think 1.2.0 is correct.

Reply:
Fixed https://github.com/autowarefoundation/autoware_adapi_msgs/pull/55/commits/7779c48a8aff5d41fdf45e91de287fb22f3a6c79